README.md

```diff
@@ -1,136 +1,138 @@
+</summary>
 
-### Translations
-
-You can also contribute by helping to translate Finamp! This is done through our Weblate instance here: <https://hosted.weblate.org/engage/finamp/>. The current translation status is this:
-
-<a href="https://hosted.weblate.org/engage/finamp/">
-  <img src="https://hosted.weblate.org/widget/finamp/finamp/horizontal-auto.svg" alt="Translation status" />
-</a>
+<img src=".">
+</details>
 
-## Known Issues
+</details>
 
-This app is still a work in progress, and has some bugs/issues that haven't been fixed yet. Here is a list of currently known issues:
 
-- Reordering the queue while shuffle is enabled is not possible at the moment. It seems like this is an issue with a dependency of Finamp (`just_audio`), and is being tracked [here](https://github.com/ryanheise/just_audio/issues/1042)
-- If you have a very large library or an older phone, performance might not be great in some places
 
-## Planned Features
+## Shoutout
+- Thanks to all the [Contributors and Maintainer](https://github.com/jmshrv/finamp/graphs/contributors) who helped to make, fix and improve Finamp! Without you Finamp wouldn't be Finamp. ❤️
+- Thanks to the [Jellyfin Contributors](https://jellyfin.org/contribute/) without whom finamp wouldn't exists in the first place and thanks for making self-hosting and privacy easier!
+- Thanks to all the Developers who created and maintain packages finamp uses too!
+- And thanks to you for using Finamp!
 
-- Improved Android Auto / Apple CarPlay support
-- Full redesign, adding more features and a home screen. See [this issue](https://github.com/jmshrv/finamp/issues/220) for more info
-- Better playlist editing
-- Multiple users/servers
-- More customization options
+______
 
-## Screenshots (Stable Version, outdated)
+<div align="center">
 
-| | |
-|:-------------------------:|:-------------------------:|
-|<img src=https://raw.githubusercontent.com/jmshrv/finamp/master/fastlane/metadata/android/en-US/images/phoneScreenshots/1.png> | <img src=https://raw.githubusercontent.com/jmshrv/finamp/master/fastlane/metadata/android/en-US/images/phoneScreenshots/2.png>
-| <img src=https://raw.githubusercontent.com/jmshrv/finamp/master/fastlane/metadata/android/en-US/images/phoneScreenshots/3.png> | <img src=https://raw.githubusercontent.com/jmshrv/finamp/master/fastlane/metadata/android/en-US/images/phoneScreenshots/4.png> |
+[<img src="app-store-badges/app-store.svg"
+    alt="Download on the App Store"
+    height="55">](https://apps.apple.com/us/app/finamp/id1574922594)
 
+[<img src="app-store-badges/fdroid.png"
+    alt="Get it on F-Droid"
+    height="80">](https://f-droid.org/packages/com.unicornsonlsd.finamp/)
 
-Name source: https://www.reddit.com/r/jellyfin/comments/hjxshn/jellyamp_crossplatform_desktop_music_player/fwqs5i0/
+[<img src="app-store-badges/play-store.png"
+    alt="Get it on Google Play"
+    height="80">](https://play.google.com/store/apps/details?id=com.unicornsonlsd.finamp)
+</div>
```

Comment:
I'd really like to move these up so people can easily get to downloading the app! We could also duplicate it at the bottom, doesn't hurt.  

And can you try to make it fit into a single row? I've seen that in many other readmes, and it looks much better. The images should have the same height too :)